Description: Fix bundler version to match debian package.
 metasploit-framework (5.0.74-parrot1) rolling-testing; urgency=medium
 .
   * Import new upstream release.
   * Rebuild package using latest bundler version.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2020-02-18

--- metasploit-framework-5.0.74.orig/Gemfile.lock
+++ metasploit-framework-5.0.74/Gemfile.lock
@@ -416,4 +416,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
